evaluator/src/test/kotlin/RuleSetTest.kt

```diff
@@ -125,22 +127,25 @@ class RuleSetTest : WordSpec() {
             "add no license errors if license is removed by license choice" {
                 val ruleSet = ruleSet(ortResult) {
                     dependencyRule("test") {
-                        licenseRule("test", LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED) {
+                        licenseRule("test", LicenseView.ONLY_CONCLUDED) {
                             require {
-                                object : RuleMatcher {
-                                    override val description = "containsLicense(license)"
-
-                                    override fun matches() = license == "LicenseRef-b".toSpdx()
-                                }
+                                +containsLicense("LicenseRef-b".toSpdx())
                             }
 
                             error(errorMessage, howToFix)
                         }
                     }
                 }
 
-                ruleSet.violations should haveSize(5)
+                ruleSet.violations should haveSize(1)
             }
         }
     }
+
+    private fun PackageRule.LicenseRule.containsLicense(expression: SpdxExpression) =
```

Comment:
Nit: The extraction of this function could have happened in a preparing commit, as it's unrelated to the actual change.

Reply:
I would prefer to leave these changes together. As with the extraction came a fix for the actual test we weren't aware of before.